packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx

```diff
@@ -22,6 +22,7 @@ export const PastElectionsListRow = ({ election }: PastElectionsListRowProps) =>
       $colLayout={PastElectionsColLayout}
       as={GhostRouterLink}
       to={generatePath(ElectionRoutes.pastElection, { id: election.id })}
+      target="_blank"
```

Comment:
This work but opening all links in a new tab isn't an ideal solution (it's inconsistent with the rest of the app). Instead could you try something like this on the `BlockLink` in the `BlockInfo` info component:
```jsx
onClick={evt => evt.stopPropagation()}
```